pkg/asset/manifests/openstack/cloudproviderconfig.go

```diff
@@ -17,6 +17,10 @@ secret-namespace = kube-system
 		res += "region = " + cloud.RegionName + "\n"
 	}
 
+	if additionalTrust {
+		res += "ca-file = /etc/pki/openshift-ca-trust/additionalTrustBundle.pem\n"
```

Comment:
additionalTrustBundle is not the same as "trust bundle for my cloud provider"

Reply:
So should the installer read the content of the file referenced from the `ca-file` option of the `clouds.yaml` file to know the cloud provider cert? We were assuming it would be passed via the `install-config.yaml`'s `additionalTrustBundle` option. I suppose both can be done.

Reply:
I thought this was the intended use case for additionalTrustBundle. What is that field supposed to be used for then?

Reply:
How are users meant to pass only the ca-cert to cloud provider? My understanding is that they would have to put that ca-cert in the additionalTrustBundle field anyway, so we may as well propagate it. Otherwise, I don't know of another means for them to pass it into the installer.

Reply:
Alternatively, this could also be a new install-config.yaml option for the [OpenStack platform](https://github.com/openshift/installer/blob/master/pkg/types/openstack/platform.go), although my preference goes to reading from the `ca-file` option of the `clouds.yaml`.